src/core_plugins/kibana/index.js

```diff
@@ -1,7 +1,13 @@
+import fs from 'fs';
+import i18nPlugin from '../i18n/server/i18n/index';
```

Comment:
Plugins shouldn't import things directly from other plugins, we'll need to expose this on the server object (as I mentioned in another comment)